fees/mellow.ts

```diff
@@ -0,0 +1,109 @@
+//  Maverick v1 fee
+import { FetchOptions, FetchResultV2, SimpleAdapter } from "../adapters/types";
+import { CHAIN } from "../helpers/chains";
+import { ChainApi } from "@defillama/sdk";
+import { httpGet } from "../utils/fetchURL";
+import { getERC4626VaultInfo } from "../helpers/erc4626";
+
+const methodology = {
+  Fees: "Fees generated from staking assets in LRT vaults.",
+  SupplySideRevenue: "Fees are distributed to supply side depositors.",
+};
+
+const MellowAbis: any = {
+  configurator: 'address:configurator',
+  priceOracle: 'address:priceOracle',
+  baseTokens: 'function baseTokens(address) view returns (address)',
+  underlyingTvl: 'function underlyingTvl() view returns (address[] tokens, uint256[] amounts)',
+  getPrice: 'function getPrice(address, address) view returns (uint256 answer, uint8 decimals)',
+  totalSupply: 'uint256:totalSupply',
+  decimals: 'uint8:decimals',
+}
+
+// get active vaults from Mellow API
+// https://points.mellow.finance/v1/vaults
+const DVstETHVault = '0x5E362eb2c0706Bd1d134689eC75176018385430B'
+const DVstETHPriceOracle = '0x39D5F9aEbBEcba99ED5d707b11d790387B5acB63'
+async function getActiveVaults(): Promise<Array<string>> {
+  return ((await httpGet('https://points.mellow.finance/v1/vaults')).map((item: any) => item.address) as Array<string>)
```

Comment:
better to use [`getConfig`](https://github.com/DefiLlama/dimension-adapters/blob/master/helpers/cache.ts#L50) here, this way, if the endpoint is not reachable we have cached data.